<launch>
	<rosparam file="launch/keyconfig.yaml" />

	<node pkg="ros_teleop" name="key_listener" type="key_listener.py"/>

	<node pkg="ros_teleop" name="key_twist" type="key_interpreter.py" output="screen">
		<param name="topic_name" value="twist"/>
	</node>

</launch>
